src/librustc_llvm/build.rs

```diff
@@ -234,6 +234,17 @@ fn main() {
         }
     }
 
+    let llvm_linker_flags = env::var_os("LLVM_LINKER_FLAGS");
```

Comment:
Could this include a small comment indicating what it's intended to be solving? (aka what link paths and such are intended to be communicated here)